crates/revm/src/batch.rs

```diff
@@ -182,15 +180,12 @@ impl BlockBatchRecord {
 
 #[cfg(test)]
 mod tests {
+    extern crate alloc;
```

Comment:
why is this required here? if we use alloc directly above

Reply:
indeed this is not needed, my bad, just fixed